pyproject.toml

```diff
@@ -7,3 +7,9 @@ requires = [
     "setuptools_scm >= 7.0.0",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.check-manifest]
+ignore = [
+    "lib/cartopy/_version.py",
+    "lib/cartopy/trace.cpp",
```

Comment:
Note that, these files are present in the `sdist`, but are not under version control, therefore require to be ignored otherwise the `check-manifest` will rightly complain that there is a mismatch.